### DISCUSSION TITLE
DietPi-Software | aMule

### DISCUSSION THREAD

**Discussion by tarrasque73** [MichaIng/DietPi#4946] [Software Request]
Hi all.

I'm new to this section / way of asking for requests, so I hope I'm not breaking any rules.

I'd like to have aMule as  a supported software in DietPi

From what I know there's no other eMule/e2k client in the supported apps, so this would "fill a hole" in a download server type installation.

aMule can run as a daemon + web interface / cmd interface so it's very well suited for a headless server.

**Comment by Joulinar** [comment]
You hit the perfect place for your request. 👍

**Comment by qfa-ok** [comment]
Hi. What happened to this? Any plans?

**Reply by Joulinar** [reply]
Nope, No ETA
